src/entities/Employee.java

```diff
@@ -0,0 +1,21 @@
+package entities;
+
+public class Employee extends User{
+    private long code;
```

Comment:
Use objects as your first approach when creating an application from scratch if you don't know the amount of data you will handle at the beginning

Reply:
Long represents an Object, and long a type of variable, so Long can be null and serialized, and all the methods that an object can have. The primitive long uses less memory space than the object Long, but since we dont know the amount of data the project will contain it is always recommended to use Java Objects